rwgame/RWGame.cpp

```diff
@@ -507,10 +507,13 @@ void RWGame::tick(float dt) {
             clockAccumulator -= 1.f;
         }
 
+        constexpr float timerClockRate = 1.f / 25.f;
```

Comment:
I don't fully understand this change yet (I'm also not aware of the surrounding code).

Where does this `/ 25.f` come from - gta3.exe?
Does it have to do with FPS or what's the story behind it?

also nit for this commit: commit title is too long; should be reworded to not break into commit description

Reply:
> Does it have to do with FPS or what's the story behind it?

Basically, yes.

In GTAIII timers are updated every frame and framerate is locked to 30fps [if you have frame limiter on], there is no accumulators or anything. They may go out of sync depending on game's uptime.